### ISSUE TITLE
Ice posts below drop down menu and stays, until clicked in an open area of the menu bar, BRAVE Browser+ other apps

### ISSUE THREAD

**Issue by macmanjimmy** [jordanbaird/Ice#394]
### Check Existing Issues

- [X] I have checked existing issues, and this issue is not a duplicate

### Description

While using the Brave Brower, the Ice Menu app activates all on its own,  not initiating the action myself. The only way to remove the menu at issue is to quick anywhere on the menu itself 

### Steps to Reproduce

1. Goto and Use Brave Browser app, in time "Ice" activates on its own, only way to remove is to click anywhere on the menu... Please see attachment 

### Ice Version

0.11.8.1

### macOS Version

15.0

### Screenshots

![Screenshot 2024-10-05 at 10 05 44 AM](https://github.com/user-attachments/assets/01301bec-d5b1-48a6-b0bb-566029dceffb)

**Comment by jordanbaird** [comment]
This is almost identical to your previous issue. Ice cannot just activate on its own. You are likely using some other app that is causing this. Please attach a screen recording of this happening. Do not open a new issue for this. Attach the video in the comments.

**Comment by macmanjimmy** [comment]
Apologies .. is the Ice menu drop-down supposed to display right under the drop-down menu path?

**Comment by jordanbaird** [comment]
Depends on what your settings are. If the Ice Bar location is set to "Dynamic", it could.

**Comment by jordanbaird** [comment]
Where is your mouse when this happens?

**Comment by macmanjimmy** [comment]
middle of the screen.. this just happened, in "Preview" curious also in the middle of the screen, not sure if normal 
![Screenshot 2024-10-06 at 1 28 51 PM](https://github.com/user-attachments/assets/eea5f405-6716-489c-ae75-37c633baa313)

**Comment by jordanbaird** [comment]
As I said, please attach a screen recording. Without seeing what exactly is happening, I can't do anything about this.

**Comment by macmanjimmy** [comment]
sure, im asking to go easy on the "Push back" , please... next time this happens will get a "screen recording" 

:-) :-) :-)

**Comment by jordanbaird** [comment]
Snarkiness? Come on. You're wasting my time by posting two nearly identical issues with next to no workable information, and you didn't follow my instructions when I asked you to post a screen recording. I'm being firm, not snarky. I have to be when I have 80+ other issues to worry about.

**Comment by macmanjimmy** [comment]
your interpretation, maybe you should get an assistant to help with all the stress :-) 

May not have noticed, but I closed the original post when I realized I posted the wrong app used..
Will reply to my single bug report, this one when the issue reoccurs 
Thanks
:-) :-) :-)

**Comment by jordanbaird** [comment]
> maybe you should get an assistant to help with all the stress

Can't exactly pay an assistant when the app is free (yes, some people pay, which I am immensely grateful for, but I don't make nearly enough for me to make a living, let alone pay someone else).

> May not have noticed, but I closed the original post when I realized I posted the wrong app used..

I would have preferred that you had kept it in the original post, and either you or I could have renamed it, but I get that. What's done is done at this point. But what I need is more information. What you're describing doesn't happen on either of my machines, so I don't even have a place to start.

**Comment by macmanjimmy** [comment]
As mentioned before..... the "ice drop-down" disappears when I go over the menu bar..
How do I initiate "screen recording" without going to the menu bar?

The issue did happen and when I executed Quicktime, the "Ice drop-down" reminded during the QUICKTIME Menu drop-down change, but when I went up to select "Record Screen" of course, as mentioned before, the "Ice Drop-Down Menu" went away 

I guess I can take a video with my phone LOL

**Comment by jordanbaird** [comment]
Shift + Command + 5 will bring up a pane of screen recording options. But I don't need the recording of what happens after it appears. I need a recording of it appearing. Your entire screen, so I can know what exactly is going on.

**Comment by jordanbaird** [comment]
Also, I just had a thought. Do you by chance use an autoclicker or some other app that controls the mouse or changes mouse events? That could very well be what's causing the problem.

**Comment by macmanjimmy** [comment]
that wasnt me, not using any app like that

"When it happens", 

happens sparadically cannot tell when it's going to happen, busily working and in the middle of the screen and pops up

**Comment by jordanbaird** [comment]
I'm sorry, but if you can't provide any more information, there really isn't anything I can do to help you. There must be something you're doing that's causing it. If you are unable to narrow it down, I can give you some things to try:

1. Turn off "Show on click"
2. Turn off "Show on scroll"
3. Remove any hotkeys you might have set up (especially the ones to show the different sections)

I'm going to close this, as out of the thousands of other users of this app, no one else has had this issue. It almost has to be something specific to your setup. If you figure anything else out, reply to this thread, and I'll reopen it.

**Comment by jordanbaird** [comment]
Also, I apologize for coming across as confrontational. As I said in an earlier comment, I sometimes need to be firm with people, as I need to be able to efficiently manage issues.

**Comment by macmanjimmy** [comment]
menu triggers with arrow is selected, thought that was the problem as well...

Been working with Apple's since 1978 and worked Tier 2 here in Austin.. 

The issue occurs with Apple Mail / Preview and Brave Browser, nothing is conflicting with my apps, I see to that... very OCPD LOL 


I'll drop it.. All in all still better than Bartender 4 which is getting sluggish under Sequoia

**Comment by macmanjimmy** [comment]
per request "Screen Record"[https://1drv.ms/v/c/c0feb8096747b2cf/Eajru6mHIVBIjiskLemU8RwB7Qv1jw7gXXq56kzA2krX6A?e=C8Wr8a](url)

**Comment by jordanbaird** [comment]
Ah, I see. You have the "Show on hover" feature enabled, correct? I can reproduce a similar behavior when "Show on hover" is enabled, and "Show on hover delay" in advanced settings is set to 0.

**Comment by macmanjimmy** [comment]
Yes sir, on both.. good guess.. JK :-) is it a setting issue? have to go back to the menu to release the drop-down

**Comment by jordanbaird** [comment]
This is definitely a bug. I went ahead and opened #401 for this with the additional details. Let's have any further discussion over there.

**Comment by macmanjimmy** [comment]
yes sir
